onecodex/lib/enums.py

```diff
@@ -28,8 +28,9 @@ class Metric(BaseEnum):
 
 class AlphaDiversityMetric(BaseEnum):
     Simpson = "simpson"
-    Chao1 = "chao1"
+    Observed_taxa = "observed_taxa"
```

Comment:
Very minor: For enums we tend to use CamelCase instead of snake_case:

```python
ObservedTaxa = "observed_taxa"
```

Reply:
CamelCase it is. Thank you.